Lib/test/test_embed.py

```diff
@@ -1331,9 +1334,9 @@ def test_init_pyvenv_cfg(self):
                                            'lib-dynload')
                 os.makedirs(lib_dynload)
             else:
-                lib_dynload = os.path.join(pyvenv_home, 'lib')
+                lib_dynload = os.path.join(pyvenv_home, 'Lib')
```

Comment:
Is the lower-case a hold-over from earlier versions of Windows?

Reply:
No, the lower case is because of the issue that I fixed in site.py that was inserting a lowercase L. Fixing that broke this test because of a case-sensitive comparison, but it has no effect on case-insensitive Windows directories (which are no longer all directories - you can mark trees as case-sensitive now - so we need to be more careful about casing).